pkg/scheduler/framework/framework.go

```diff
@@ -366,18 +366,23 @@ var markUnscheduledForAndUpdate = func(ctx context.Context, hubClient client.Cli
 	// unscheduledBinding from "scheduled" to "bound".
 	binding.Spec.State = placementv1beta1.BindingStateUnscheduled
 	err := hubClient.Update(ctx, binding, &client.UpdateOptions{})
-	klog.V(2).InfoS("Marking binding as unscheduled", "clusterResourceBinding", klog.KObj(binding), "error", err)
+	if err == nil {
```

Comment:
where is the err case going to be logged?

Reply:
Where updateBinding is called,

- https://github.com/Azure/fleet/pull/924/files#diff-fc2ffba6d515e0ea81ed25218cc712a5949e0178a15fe3ebc6274d7ee27f401eR299
- https://github.com/Azure/fleet/pull/924/files#diff-fc2ffba6d515e0ea81ed25218cc712a5949e0178a15fe3ebc6274d7ee27f401eR305
- https://github.com/Azure/fleet/pull/924/files#diff-fc2ffba6d515e0ea81ed25218cc712a5949e0178a15fe3ebc6274d7ee27f401eR686
- https://github.com/Azure/fleet/pull/924/files#diff-fc2ffba6d515e0ea81ed25218cc712a5949e0178a15fe3ebc6274d7ee27f401eR839